virtcontainers/sandbox.go

```diff
@@ -1283,15 +1284,19 @@ func (s *Sandbox) CreateContainer(contConfig ContainerConfig) (VCContainer, erro
 		return nil, err
 	}
 
-	// Store it.
-	err = c.storeContainer()
+	// Sandbox is reponsable to update VM resources needed by Containers
+	s.updateResources()
 	if err != nil {
 		return nil, err
 	}
 
-	// Update sandbox config.
-	s.config.Containers = append(s.config.Containers, contConfig)
-	err = s.storage.storeSandboxResource(s.id, configFileType, *(s.config))
```

Comment:
Why did you remove those two lines? We still want to store the resources from a sandbox perspective, right?

Reply:
They are moved to api.go but @jcvenegas please keep them in sandbox.go so that api.go doesn't need to call any storage related functions (as of now).